src/main/java/OperationFactory.java

```diff
@@ -1,2 +1,37 @@
 public class OperationFactory {
```

Comment:
The idea of implement operation factory is great. It leaves space for further development.